common/config/root.go

```diff
@@ -135,15 +133,15 @@ func isSignalExists(dest *odigosv1.Destination, signal common.ObservabilitySigna
 	return false
 }
 
-func isTracingEnabled(dest *odigosv1.Destination) bool {
+func isTracingEnabled(dest ExporterConfigurer) bool {
```

Comment:
nit: now that we have a common interface `SignalSpecific` we can potentially use it and share this code between destinations and processors.

Reply:
I actually had it in the next dev branch. Pushed it here.